remote/src/main/java/com/kdjj/remote/dao/FirebaseStorageServiceImpl.kt

```diff
@@ -16,13 +17,12 @@ internal class FirebaseStorageServiceImpl @Inject constructor(
         uri: String
     ): Result<ByteArray> {
         return withContext(Dispatchers.IO) {
-            try {
-                val byteArray = storageRef.storage
+            runCatching {
+                storageRef.storage
                     .getReferenceFromUrl(uri)
                     .getBytes(MAX_SIZE).await()
```

Comment:
체인 사용할 때 한칸 띄는게 좋아보입니다! 수정 부탁드립니다.
```suggestion
            runCatching {
                storageRef.storage
                    .getReferenceFromUrl(uri)
                    .getBytes(MAX_SIZE)
                    .await()
```

Reply:
수정했습니다~